components/Breadcrumb/src/index.scss

```diff
@@ -23,11 +37,11 @@ ol.denhaag-breadcrumb__list {
 }
 
 .denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child) {
-  --denhaag-breadcrumb-background-color: var(--denhaag-breadcrumb-dots-background-color, inherit);
+  --denhaag-breadcrumb-link-background-color: var(--denhaag-breadcrumb-dots-background-color, inherit);
```

Comment:
`inherit` is not necessary, because the default fallback is `inherit` when the value could not be read.

Reply:
This logic has been approved before, currently only the name has been changed. 

Therefor I'll not change the logic. Overwritting an existing CSS variable with a new one requires a back-up variable. Otherwise it's possible to be undefined, therefor the whole variable we cast to is also undefined.